README.md

```diff
@@ -1,6 +1,34 @@
 # hmpps-template-typescript
 Template github repo used for new Typescript based projects.
 
+# Instructions
+
+If this is a Digital Prison Services project then the project will be created as part of bootstrapping - 
+see https://github.com/ministryofjustice/dps-project-bootstrap.
+
+## Creating a CloudPlatform namespace
+
+When deploying to a new namespace, you may wish to use this template typescript project namespace as the basis for your new namespace:
+
+<https://github.com/ministryofjustice/cloud-platform-environments/tree/main/namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-template-typescript>
+
+This template namespace includes an AWS elasticache setup - which is required by this template project.
+
+Copy this folder, update all the existing namespace references, and submit a PR to the CloudPlatform team. Further instructions from the CloudPlatform team can be found here: <https://user-guide.cloud-platform.service.justice.gov.uk/#cloud-platform-user-guide>
+
+## Renaming from HMPPS Template Kotlin - github Actions
```

Comment:
reference to kotlin

this doesn't have github actions enabled

Reply:
fixed